deployments/infrastructure-target-aws.nix

```diff
@@ -33,11 +39,54 @@ rec {
     };
 
     deployment.ec2 = {
-      accessKeyId = IOHKaccessKeyId;
-
+      inherit accessKeyId;
+      elasticIPv4 = resources.elasticIPs.cardanod-ip;
       instanceType = mkForce "r3.2xlarge";
       ebsInitialRootDiskSize = mkForce 50;
       associatePublicIpAddress = true;
+      securityGroups = [
+        resources.ec2SecurityGroups."allow-all-ssh-${region}-${org}"
+      ];
+    };
+  };
+
+  resources = {
+    elasticIPs = {
+      hydra-ip    = { inherit region accessKeyId; };
```

Comment:
These should still be in production config

Reply:
..but it would break the AWS separation!

Reply:
I'm not following, production is only on AWS. There won't be any other productions :)

Reply:
So why having AWS separate at all?

Reply:
That's part of my confusion, actually, @domenkozar..  you used to say that we want to keep AWS-specific configuration (aka physical spec) separate.

Reply:
Yes, it's for deploying testing changes for AWS.

Reply:
..and this means the consistent way would be to have `infrastructure-env-production-target-aws.nix`

Reply:
That would imply there are other (can exist) production deployment backends, which is false.

Reply:
@domenkozar, then why isn't the very same logic applicable to the cluster itself?

Please, don't get alarmed by my questions -- they are the root reason why I started folding `*-target-aws.nix` -- I'm not understanding the difference between approaches here -- and I thought we wanted a common approach..

Reply:
One possibility is that we want to be potentially able to distribute the cluster to GCE/Azure/whatever.

That I can understand.

But from this _might_ follow a requirement that we keep a deployer there as well -- at least for performance reasons.

Hence a non-AWS deployer could potentially make sense.